lib/scss/stanford_events.views.scss

```diff
@@ -0,0 +1,225 @@
+
+    @include grid-media-max('sm') {
+      align-items: center;
+      display: flex;
+    }
+
+    max-width: 45px;
+
+    span {
+      display: block;
+    }
+
+    .su-event-start-month,
+    .su-event-end-month {
+      @include modular-typography(0);
+
+      color: $su-color-black;
+      font-weight: $su-font-bold;
+      text-transform: uppercase;
+    }
+
+    .su-event-list-item__date__seperator {
+      @include small-paragraph;
+      @include modular-spacing('margin', null null 0 null);
+      color: $su-color-black;
+
+      @include grid-media-max('sm') {
+        @include modular-spacing(margin, null -1);
+        min-width: 24px;
+      }
+    }
+
+    .su-event-start-date,
+    .su-event-end-date {
+      @include modular-typography(4);
+      color: $su-color-black;
+      font-weight: $su-font-bold;
+    }
+  }
+
+  .su-event-list-item__details {
+    @include big-paragraph;
+
+    color: $su-color-black;
+    flex-grow: 2;
+
+    .event-list-item__audience {
+      @include modular-spacing('margin', null null -2 null);
+
+      a {
+        color: $su-color-black;
+        font-weight: $su-font-regular;
+        text-decoration: none;
+      }
+    }
+
+    h2 {
+      @include modular-typography(3);
+      @include modular-spacing('margin', null null -1 null);
+
+      > a {
+        color: $su-color-bright-red;
+        font-weight: $su-font-bold;
+        text-decoration: none;
+
+        &:hover,
+        &:focus,
+        &:active {
+          color: $su-color-black;
+        }
+
+        &::after {
+          background-color: $su-color-bright-red;
+          height: 20px;
+          width: 20px;
+        }
+
+      }
+    }
+
+    .views-field-su-event-subheadline {
+      @include modular-spacing('margin', null null 1 null);
+    }
+
+    .views-field-su-event-dek {
+      @include modular-spacing('margin', null null 0 null);
+    }
+
+    .views-field-su-event-date-time {
+      @include small-paragraph;
+      @include modular-spacing('margin', null null -2 null);
+
+      // Align multirow text with the icon.
+      display: flex;
+
+      span {
+        &::before {
+          background-color: $su-color-black;
+        }
+      }
+    }
+
+    .address {
+      @include small-paragraph;
+    }
+
+    .event-list-item__location,
+    .views-field-su-event-location {
+      @include small-paragraph;
+      @include modular-spacing('margin', null null 3 null);
+
+      // Align multirow text with the icon.
+      display: flex;
+
+      span {
+        &::before {
+          background-color: $su-color-black;
+        }
+      }
+    }
+  }
+}
+
+// Wrapper spacing
+.views-infinite-scroll-content-wrapper {
+  @include modular-spacing('margin-bottom', 5);
+
+  & + .js-pager__items {
+    @include modular-spacing('margin-bottom', 8);
+  }
+}
+
+// Load more button.
+.js-pager__items {
+  text-align: center;
+
+  .pager__item {
+    display: inline-block;
+
+    .button {
+      @include button-big;
+    }
+  }
+}
```

Comment:
@jenbreese Can we add specificity to these statements so they are not affecting all pages on the page when this css file is loaded but rather tie them to the views we want. There should be css classes on the views already but if we need to add more we can.